include/z64actor.h

```diff
@@ -127,52 +126,42 @@ typedef struct Actor {
+    /* 0x090 */ f32     xzDistFromLink;
+    /* 0x094 */ f32     yDistFromLink;
     /* 0x098 */ CollisionCheckInfo colChkInfo;
     /* 0x0B4 */ ActorShape shape;
     /* 0x0CC */ Vec3f   unk_CC[2];
-    /* 0x0E4 */ Vec3f   unk_E4; // Stores result of some vector transformation involving actor xyz vector, and a matrix at Global Context + 11D60
+    /* 0x0E4 */ Vec3f   unk_E4; // Stores result of some vector transformation involving actor xyz vector and mf_11D60
     /* 0x0F0 */ f32     unk_F0; // Related to above
     /* 0x0F4 */ f32     unk_F4;
     /* 0x0F8 */ f32     unk_F8;
     /* 0x0FC */ f32     unk_FC;
-    /* 0x100 */ Vec3f   pos4; // Final Coordinates last frame (collision, NTSC 1.0 f 8002F8E0)
+    /* 0x100 */ Vec3f   pos4;
     /* 0x10C */ u8      unk_10C; // Z-Target related
     /* 0x10D */ u8      unk_10D; // Z-Target related
-    /* 0x10E */ u16     textId; // Text id to pass to link/display when interacting with an actor (navi text, probably others)
-    /* 0x110 */ u16     freeze; // Used for the "Redead Freeze" attack. Also used in func_80059EC8
-    /* 0x112 */ u16     unk_112; // Damage color effect, first 12 bits controls color which can only be blue red and white, last 4 bits unknown, can't be 0
-    /* 0x114 */ u8      unk_114; // Damage color effect timer, decremented toward 0 every frame
-    /* 0x115 */ u8      activelyDrawn; // Indicates whether the actor is currently being drawn (but not through lens). 01 for yes, 00 for no
-    /* 0x116 */ u8      unk_116; // Set within a routine that deals with collision
+    /* 0x10E */ u16     textId; // Text id to pass to link/display when interacting with an actor
+    /* 0x110 */ u16     freeze;
+    /* 0x112 */ u16     dmgEffectParams; // Specifies damage effect color (white/red/blue) and if opaque or translucent
+    /* 0x114 */ u8      dmgEffectTimer;
+    /* 0x115 */ u8      activelyDrawn; // Indicates whether the actor is currently being drawn (but not through lens)
+    /* 0x116 */ u8      unk_116;
     /* 0x117 */ u8      naviEnemyId; // Sets what 0600 dialog to display when talking to navi. Default 0xFF
-
-    /* 0x118 */ struct Actor* attachedA; // Interfacing Actor?
-    // e.g. Link holding chu, Chu instance stores ptr to Link instance here;
-    //      Anju having Link's ptr when giving an item
-    //      Volvagia Hole stores Volvagia Flying here
-
-    /* 0x11C */ struct Actor* attachedB; // Attached to Actor
-    // e.g. Link holding chu, Link instance stores ptr to Bombchu instance here
-
+    /* 0x118 */ struct Actor* attachedA; // Attached By?
+    /* 0x11C */ struct Actor* attachedB; // Attached To?
     /* 0x120 */ struct Actor* prev; // Previous Actor of this type
     /* 0x124 */ struct Actor* next; // Next Actor of this type
-
-    /* 0x128 */ ActorFunc init; // Initialization Routine. Mandatory
+    /* 0x128 */ ActorFunc init; // Initialization Routine. All actors must have an init function.
     /* 0x12C */ ActorFunc destroy; // Destruction Routine
```

Comment:
Maybe add where these are called too for consistency? `Actor_Init` and `Actor_UpdateAll` for `init` and `Actor_Destroy` for `destroy`. Rest looks good.